pyvista/utilities/helpers.py

```diff
@@ -1458,3 +1459,56 @@ def cubemap(path='', prefix='', ext='.jpg'):
         texture.SetInputDataObject(i, flip.GetOutput())
 
     return texture
+
+
+def set_default_active_vectors(mesh: 'pyvista.DataSet') -> None:
+    """Set a default vectors on mesh, if not already set.
+
+    If an active vector already exists, no changes are made.
+
+    If an active vectors does not exist, it checks for possibly cell
+    or point arrays with shape ``(n, 3)``.  If only one exists, then
+    it is set as the active vectors.  If none or more than one exists,
+    then a ValueError is raised.
+
+    Parameters
+    ----------
+    mesh : pyvista.DataSet
+        Dataset to set default active vectors.
+
+     Raises
+    ------
+    MissingDataError
+        If no vector-like arrays exist.
+    AmbiguousDataError
+        If more than one vector-like arrays exist.
+
+    """
+    if mesh.active_vectors_name is not None:
+        return
+
+    point_data = mesh.point_data
+    cell_data = mesh.cell_data
+
+    possible_vectors_point = [
+        name for name, value in point_data.items() if value.ndim == 2 and value.shape[1] == 3
+    ]
+    possible_vectors_cell = [
+        name for name, value in cell_data.items() if value.ndim == 2 and value.shape[1] == 3
+    ]
+
+    possible_vectors = possible_vectors_point + possible_vectors_cell
+    n_possible_vectors = len(possible_vectors_point) + len(possible_vectors_cell)
+
+    if n_possible_vectors == 1:
+        if len(possible_vectors_point) == 1:
+            mesh.set_active_vectors(possible_vectors_point[0], preference='point')
+        else:
+            mesh.set_active_vectors(possible_vectors_cell[0], preference='cell')
```

Comment:
Might be a bit simpler
```suggestion
        if len(possible_vectors_point) == 1:
            preference = 'point'
        else:
            preference = 'cell'
        mesh.set_active_vectors(possible_vectors[0], preference=preference)
```